task-sdk/src/airflow/sdk/execution_time/context.py

```diff
@@ -374,6 +374,7 @@ def _get_asset_from_db(name: str | None = None, uri: str | None = None) -> Asset
             ToSupervisor,
         )
         from airflow.sdk.execution_time.task_runner import SUPERVISOR_COMMS
+        from airflow.serialization.serde import deserialize
```

Comment:
We are not allowed to use non-sdk modules in sdk code. Why does this need to use `deserialize` (and `serialize` below)?